licenses/management/commands/publish.py

```diff
@@ -0,0 +1,172 @@
+        raise CommandError(e)
+    output_dir = os.path.abspath(os.path.expanduser(output_dir))
+    if os.path.isdir(output_dir):
+        rmtree(output_dir)
+        os.makedirs(output_dir)
+    else:
+        os.makedirs(output_dir)
+    try:
+        render_to_dir(output_dir, urls_to_distill, stdout)
+    except DistillError as err:
+        raise CommandError(str(err)) from err
+
+
+def check_if_build_to_push(branch: str):
+    """Navigates to a branch and checks the status of the branch
+
+    Returns:
+        True if files need to be committed
+        False if the working tree is clean
+    """
+    subprocess.run(
+        ["git", "checkout", f"{branch}"], cwd=TRANSLATION_REPOSITORY_DIRECTORY
+    )
+    status = (
+        subprocess.check_output(["git", "status"], cwd=TRANSLATION_REPOSITORY_DIRECTORY)
+        .decode()
+        .splitlines()
+    )
+    if "nothing to commit, working tree clean" in status:
+        return False
+    return True
+
+
+def git_on_branch_and_pull(branch: str):
+    """Commands to checkout and pull from a branch"""
+    subprocess.run(
+        ["git", "checkout", f"{branch}"], cwd=TRANSLATION_REPOSITORY_DIRECTORY
+    )
+    return subprocess.run(
+        ["git", "pull", "origin", f"{branch}"], cwd=TRANSLATION_REPOSITORY_DIRECTORY
+    )
+
+
+def git_commit_and_push(branch: str):
+    """Command to git checkout, commit, and push branch"""
+    subprocess.run(["git", "add", "build/"], cwd=TRANSLATION_REPOSITORY_DIRECTORY)
+    subprocess.run(
+        ["git", "commit", "-m", f"{branch}"], cwd=TRANSLATION_REPOSITORY_DIRECTORY
+    )
+    return subprocess.run(
+        ["git", "push", "origin", f"{branch}"], cwd=TRANSLATION_REPOSITORY_DIRECTORY
+    )
+
+
+def pull_translations_branches():
+    """Git pulls branches in cc-licenses-data to update local git registry"""
+    subprocess.run(["git", "checkout", "develop"], cwd=TRANSLATION_REPOSITORY_DIRECTORY)
+    return subprocess.run(["git", "pull"], cwd=TRANSLATION_REPOSITORY_DIRECTORY)
+
+
+def list_open_branches():
+    """List of open branches in cc-licenses-data repo
+    """
+    pull_translations_branches()
+    branches = (
+        subprocess.check_output(
+            ["git", "branch", "--list"], cwd=TRANSLATION_REPOSITORY_DIRECTORY
+        )
+        .decode()
+        .split("\n")
+    )
+    print("\n\nWhich branch are we publishing to?\n")
+    for b in branches:
+        print(b)
+
+
+def publish_branch(obj: object, branch: str):
+    """Workflow for publishing a single branch"""
+    git_on_branch_and_pull(branch)
+    run_django_distill(obj)
+    build_to_push = check_if_build_to_push(branch)
+    if build_to_push:
+        git_commit_and_push(branch)
+    else:
+        print(f"\n{branch} build dir is up to date.\n")
+
+
+def publish_all(obj: object):
+    """Workflow for checking branches other than develop and updating their build dir
+
+    Develop is not checked because it serves as the source of truth. It should be
```

Comment:
Was it develop, or master, that was going to be the "official" site?

Reply:
Actually I think this comment is out of date anyway, we're not skipping develop anymore.